data/scripts/field_move_scripts.inc

```diff
@@ -1,332 +1,332 @@
-PetalburgWoods_EventScript_2906BB:: @ 82906BB
-Route103_EventScript_2906BB:: @ 82906BB
-Route104_EventScript_2906BB:: @ 82906BB
-Route110_TrickHousePuzzle1_EventScript_2906BB:: @ 82906BB
-Route111_EventScript_2906BB:: @ 82906BB
-Route116_EventScript_2906BB:: @ 82906BB
-Route117_EventScript_2906BB:: @ 82906BB
-Route118_EventScript_2906BB:: @ 82906BB
-Route120_EventScript_2906BB:: @ 82906BB
-Route121_EventScript_2906BB:: @ 82906BB
-Route123_EventScript_2906BB:: @ 82906BB
+PetalburgWoods_EventScript_CutTree:: @ 82906BB
+Route103_EventScript_CutTree:: @ 82906BB
+Route104_EventScript_CutTree:: @ 82906BB
+Route110_TrickHousePuzzle1_EventScript_CutTree:: @ 82906BB
+Route111_EventScript_CutTree:: @ 82906BB
+Route116_EventScript_CutTree:: @ 82906BB
+Route117_EventScript_CutTree:: @ 82906BB
+Route118_EventScript_CutTree:: @ 82906BB
+Route120_EventScript_CutTree:: @ 82906BB
+Route121_EventScript_CutTree:: @ 82906BB
+Route123_EventScript_CutTree:: @ 82906BB
 	lockall
-	goto_if_unset FLAG_BADGE01_GET, Route103_EventScript_290721
+	goto_if_unset FLAG_BADGE01_GET, Route103_EventScript_CheckTreeCantCut
```

Comment:
These scripts aren't specific to `Route103`. The same is true for most of the other changes in this file.